tracer/src/Datadog.Trace/Debugger/LiveDebugger.cs

```diff
@@ -280,6 +284,89 @@ internal void UpdateAddedProbeInstrumentations(IReadOnlyList<ProbeDefinition> ad
             }
         }
 
+        private static string[] SafeParseSignature(string signature)
```

Comment:
Are these only _types_, or method signatures?

Reply:
the `signature` is only the fully qualified names of the types, comma separated and surrounded by `(` and `)`